Libraries/Components/TextInput/TextInput.js

```diff
@@ -155,6 +155,15 @@ export type KeyboardType =
   // Android-only
   | 'visible-password';
 
+export type InputMode =
+  | 'text'
+  | 'decimal'
+  | 'numeric'
+  | 'tel'
+  | 'search'
+  | 'email'
+  | 'url';
```

Comment:
'none' should be added to the allowed values even if we do nothing with it

Reply:
Sounds good, I've just added `none` there and mapped it to `default`